### PR TITLE
Cspl 1753 manual int test workflow

### PR DESCRIPTION
**Pull request by pdhanoyasplunk**
Added Manual Int test workflow. 
The workflow can be triggered with operator in namespace or cluster-wide scope.

### CODE REVIEW

File: .github/workflows/manual-int-test-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      TEST_CLUSTER_PLATFORM: eks
+      EKS_VPC_PRIVATE_SUBNET_STRING: ${{ secrets.EKS_VPC_PRIVATE_SUBNET_STRING }}
+      EKS_VPC_PUBLIC_SUBNET_STRING: ${{ secrets.EKS_VPC_PUBLIC_SUBNET_STRING }}
+      TEST_BUCKET: ${{ secrets.TEST_BUCKET }}
+      TEST_INDEXES_S3_BUCKET: ${{ secrets.TEST_INDEXES_S3_BUCKET }}
+      ECR_REPOSITORY: ${{ secrets.ECR_REPOSITORY }}
+      PRIVATE_REGISTRY: ${{ secrets.ECR_REPOSITORY }}
+      S3_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+      ENTERPRISE_LICENSE_LOCATION: ${{ secrets.ENTERPRISE_LICENSE_LOCATION }}
+      CLUSTER_WIDE: ${{ github.event.inputs.release_version }}
```
Comment (jambrosiano) [inline .github/workflows/manual-int-test-workflow.yml:45]: Shouldn't CLUSTER_WIDE be set to the value grabbed on lines 5-8 block? What has release_version to do with it?

File: .github/workflows/manual-int-test-workflow.yml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      - name: Set Test Cluster Name
+        run: |
+          echo "TEST_CLUSTER_NAME=eks-integration-test-cluster-${{ matrix.test }}-$GITHUB_RUN_ID" >> $GITHUB_ENV
+      - name: Set Test Cluster Nodes and Parallel Runs
+        run: >-
+          if grep -q "appframework" <<< "${{ matrix.test }}"; then
+            echo "CLUSTER_WORKERS=5" >> $GITHUB_ENV
+            echo "CLUSTER_NODES=2" >> $GITHUB_ENV  
+          fi
+      - name: Checkcout code
```
Comment (jambrosiano) [inline .github/workflows/manual-int-test-workflow.yml:56]: nit: "Checkout" typo

File: .github/workflows/manual-int-test-workflow.yml (hunk 1 above)
Comment (pdhanoyasplunk) [reply .github/workflows/manual-int-test-workflow.yml]: It was left by mistake. Corrected.

File: .github/workflows/manual-int-test-workflow.yml (hunk 2 above)
Comment (pdhanoyasplunk) [reply .github/workflows/manual-int-test-workflow.yml]: Corrected.

**Pull request closed without merging**